libs/output-mapping/src/TableLoader.php

```diff
@@ -0,0 +1,242 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\OutputMapping;
+
+use Keboola\Csv\CsvFile;
+use Keboola\OutputMapping\DeferredTasks\LoadTableQueue;
+use Keboola\OutputMapping\DeferredTasks\LoadTableTaskInterface;
+use Keboola\OutputMapping\DeferredTasks\TableWriter\CreateAndLoadTableTask;
+use Keboola\OutputMapping\DeferredTasks\TableWriter\LoadTableTask;
+use Keboola\OutputMapping\Exception\InvalidOutputException;
+use Keboola\OutputMapping\Mapping\MappingFromProcessedConfiguration;
+use Keboola\OutputMapping\Mapping\MappingFromRawConfigurationAndPhysicalDataWithManifest;
+use Keboola\OutputMapping\Mapping\MappingStorageSources;
+use Keboola\OutputMapping\Staging\StrategyFactory;
+use Keboola\OutputMapping\Storage\StoragePreparer;
+use Keboola\OutputMapping\Writer\Table\BranchResolver;
+use Keboola\OutputMapping\Writer\Table\MappingDestination;
+use Keboola\OutputMapping\Writer\Table\MetadataSetter;
+use Keboola\OutputMapping\Writer\Table\SlicerDecider;
+use Keboola\OutputMapping\Writer\Table\StrategyInterfaceNew;
+use Keboola\OutputMapping\Writer\Table\TableConfigurationResolverNew;
+use Keboola\OutputMapping\Writer\Table\TableConfigurationValidator;
+use Keboola\OutputMapping\Writer\Table\TableDefinition\TableDefinition;
+use Keboola\OutputMapping\Writer\Table\TableDefinition\TableDefinitionFactory;
+use Keboola\StorageApi\ClientException;
+use Keboola\StorageApiBranch\ClientWrapper;
+use Keboola\Temp\Temp;
+use Psr\Log\LoggerInterface;
+use Throwable;
+
+class TableLoader
```

Comment:
toto beru, že necháme bez testů, protože to je otestovaný těma funkcionálníma testama